index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <title>Awesome Books</title>
+  </head>
+  <body>
+    <main>
+      <h1 class="heading">Awesome books</h1>
+      <div class="all-books">
+          <!-- <p class="title">Book1</p>
+          <p class="author">Author1</p>
+          <button class="remove">Remove</button>
+          <hr /> -->
```

Comment:
- [ ] Kindly remove all commented code from your project, you can leave comments about your code but not commented code, this will make your codebase look neat and readable 😉